config.go

```diff
@@ -63,6 +63,12 @@ type BurrowConfig struct {
 		ZookeeperPort int      `gcfg:"zookeeper-port"`
 		ZookeeperPath []string `gcfg:"zookeeper-path"`
 	}
+	Druid map[string]*struct {
+		Zookeepers    []string `gcfg:"zookeeper"`
```

Comment:
Right, zookeeper does nothing for Druid. I left it just in case we like to do something similar to storm accessing zookeeper directory to get consumer info. But we don't need it for now. If we are comfortable getting offset just from druid supervisor, I'll remove zookeeper from druid.